bluepyopt/tests/adexp_opt.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
```

Comment:
is this compatible with the nosetests?

Reply:
In principle yet, in practice probably not. I will look into how to do this.

Reply:
ok, will nosetests automatically pick up these filenames? (not sure, but they might have to start with test_ ?)

Reply:
Okay I will look into this tomorrow.

Reply:
file names changed.